docs/book/src/topics/managedcluster.md

```diff
@@ -102,7 +102,8 @@ spec:
   version: v1.21.2
   networkPolicy: azure # or calico
   networkPlugin: azure # or kubenet
-  sku: Free # or Paid
+  sku:
+    tier: Free # or Paid
```

Comment:
is this just a documentation issue or has the API changed? I don't see it anywhere in our example template

Reply:
it's been that way since it was added: https://github.com/kubernetes-sigs/cluster-api-provider-azure/commit/f5a79aa1b3187507db7eafc15446a671594273f6

looks like it was added erroneously in https://github.com/kubernetes-sigs/cluster-api-provider-azure/commit/1963a3265fe30c654c6e09eead1da65bc3575e93